pixeltable/metadata/__init__.py

```diff
@@ -55,6 +55,11 @@ def upgrade_md(engine: sql.engine.Engine) -> None:
         system_info = session.query(SystemInfo).one().md
         md_version = system_info['schema_version']
         assert isinstance(md_version, int)
+        if md_version > VERSION:
+            raise RuntimeError(
+                f'Metadata version {md_version} is newer than current version {VERSION}. '
```

Comment:
I'd argue this is not going to be a meaningful error message for users, since the MD version is an internal recordkeeping device that most users won't be (and shouldn't need to be) aware of, and to a user it's easily confused with "pixeltable version".

Something like this would be clearer:

```
"This Pixeltable database was created with a newer Pixeltable version"
" than the one currently installed (v{pxt.__version__}).\n"
"Please update to the latest Pixeltable version by running: pip install --upgrade pixeltable"
```
It'd still be useful to write the MD version information to a `_logger.error()` call.